dhctl/cmd/dhctl/main.go

```diff
@@ -40,6 +41,299 @@ import (
+			},
+			Parrent: "bootstrap-phase",
+		},
+		{
+			Name: "converge",
+			Help: "Converge kubernetes cluster.",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineConvergeCommand(cmd)
+			},
+		},
+		{
+			Name: "converge-periodical",
+			Help: "Start service for periodical run converge.",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineConvergeCommand(cmd)
+			},
+		},
+		{
+			Name: "lock",
+			Help: "Converge cluster lock",
+		},
+		{
+			Name: "release",
+			Help: "Release converge lock fully. It's remove converge lease lock from cluster regardless of owner. Be careful",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineReleaseConvergeLockCommand(cmd)
+			},
+			Parrent: "lock",
+		},
+		{
+			Name: "destroy",
+			Help: "Destroy Kubernetes cluster.",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineDestroyCommand(cmd)
+			},
+		},
+		{
+			Name: "terraform",
+			Help: "Terraform commands.",
+		},
+		{
+			Name: "converge-exporter",
+			Help: "Run terraform converge exporter.",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineTerraformConvergeExporterCommand(cmd)
+			},
+			Parrent: "terraform",
+		},
+		{
+			Name: "check",
+			Help: "Check differences between state of Kubernetes cluster and Terraform state.",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineTerraformCheckCommand(cmd)
+			},
+			Parrent: "terraform",
+		},
+		{
+			Name: "config",
+			Help: "Load, edit and save various dhctl configurations.",
+		},
+		{
+			Name:    "parse",
+			Help:    "Parse, validate and output configurations.",
+			Parrent: "config",
+		},
+		{
+			Name: "cluster-configuration",
+			Help: "Parse configuration and print it.",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineCommandParseClusterConfiguration(cmd)
+			},
+			Parrent: "parse",
+		},
+		{
+			Name: "cloud-discovery-data",
+			Help: "Parse cloud discovery data and print it.",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineCommandParseCloudDiscoveryData(cmd)
+			},
+			Parrent: "parse",
+		},
+		{
+			Name:    "render",
+			Help:    "Render transitional configurations.",
+			Parrent: "config",
+		},
+		{
+			Name: "bashible-bundle",
+			Help: "Render bashible bundle.",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineRenderBashibleBundle(cmd)
+			},
+			Parrent: "render",
+		},
+		{
+			Name: "kubeadm-config",
+			Help: "Render kubeadm config.",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineRenderKubeadmConfig(cmd)
+			},
+			Parrent: "render",
+		},
+		{
+			Name: "master-bootstrap-scripts",
+			Help: "Render master bootstrap scripts.",
+			DefineFunc: func(cmd *kingpin.CmdClause) {
+				commands.DefineRenderMasterBootstrap(cmd)
+			},
+			Parrent: "render",
+		},
+		{
+			Name:    "edit",
+			Help:    "Change configuration files in Kubernetes cluster conveniently and safely.",
+			Parrent: "config",
```

Comment:
can we add DefineFunc for this command? and remove extra condition below:

https://github.com/deckhouse/deckhouse/pull/11007/files#diff-40e9733c1ba7d60d4d71b4d32bf6c1bb096de2a87868831ec09bf6d7dc2953ccR639-R643